deploy/cloud/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Sealos cloud cluster image
+
+## usage
+
+### prepare
+
+1. A cloud domain and dns to your k8s cluster. Suppose your domain name is `cloud.example.io`.
+2. A TLS cert that resolves `cloud.example.io` and `*.cloud.example.io`
+
+### save your cert file to a sealos config file
+
+```yaml
+# tls-secret.yaml
+apiVersion: apps.sealos.io/v1beta1
+kind: Config
+metadata:
+  name: secret
+spec:
+  path: manifests/tls-secret.yaml
+  match: ghcr.io/labring/sealos-cloud:dev
+  strategy: merge
+  data: |
+    data:
+      tls.crt: <your-tls.crt-base64>
+      tls.key: <your-tls.key-base64>
+```
+
+### run sealos cloud cluster image
+```shell
+sealos run ghcr.io/labring/sealos-cloud:dev --config-file tls-secret.yaml --env cloudDomain="cloud.example.com"
+```
```

Comment:
How to cname or use `/etc/hosts` in some POC demo case?

Reply:
Due to the current architecture design and code reasons, deployment without a valid certificate is not supported.

Reply:
`cname` and/or `etc/hosts` is controlled by outside handlers such as `godaddy`/`alidns` etc, AIO script can do nothing but a deployment pre-requirement doc, with examples how to cname domain and how to obtain free certs.